src/main/java/org/telegram/bot/engine/ImageSenderBot.java

```diff
@@ -0,0 +1,109 @@
+package org.telegram.bot.engine;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.api.methods.send.SendPhoto;
+import org.telegram.telegrambots.meta.api.objects.InputFile;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;
+
```

Comment:
This thing should be split into 2 classes: high-level bot functionality, like managing subscriptions, and org.telegram.telegrambots adapter. Easy to say, hard to do.
Possible solutions:
- callbacks
- observer
- command queue with polling